tokio-trace/src/span.rs

```diff
@@ -154,18 +142,11 @@ use {
 /// span will silently do nothing. Thus, the handle can be used in the same
 /// manner regardless of whether or not the trace is currently being collected.
 #[derive(Clone, PartialEq, Hash)]
-pub struct Span<'a> {
+pub struct Span {
```

Comment:
this means we can drop the lifetime from `tokio_trace_futures::Instrumented`?

Reply:
Yes; in fact, this will break the current `Instrumented`.

Reply:
Works for me! its an easy upgrade